tests/integration/upgrades/test_small_deployment_upgrades.py

```diff
@@ -61,6 +63,39 @@
 #  Auxiliary functions
 #
 #######################################################################
+async def _refresh(
```

Comment:
maybe externalise it in a helper since all other upgrade related tests will benefit from it